backend/django_core/apps/user/migrations/0010_alter_customuser_username.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 5.0.3 on 2024-04-05 06:54
+
+import apps.user.validators.username
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("user", "0009_merge_20240317_1131"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="customuser",
+            name="username",
+            field=models.CharField(
+                error_messages={"unique": "A user with that username already exists."},
+                help_text="Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.",
+                max_length=150,
+                unique=True,
+                validators=[
+                    apps.user.validators.username.username_validator,
+                    django.core.validators.RegexValidator(
+                        "^[a-zA-Z0-9_-]+#[0-9]{4}$",
+                        message="Username is not valid for this regex `^[a-zA-Z0-9_-]+#[0-9]{4}$`",
+                    ),
+                ],
+                verbose_name="username",
+            ),
+        ),
+    ]
```

Comment:
This is a local file, it should not be comitted

Reply:
https://github.com/coreproject-moe/CoreProject-Monorepo/blob/fix/comment-responsive-logic/backend/django_core/apps/user/migrations/0010_alter_customuser_username.py
This is committed by you.

See migrations are being comment.

Reply:
Fun fact is i dont see this migration on my system lol